lib/auth/v2/queryAuthCheck.js

```diff
@@ -34,8 +34,9 @@ function check(request, log, data) {
     }
 
     const currentTime = Date.now();
+    const preSignUrlExpiry = process.env.PRE_SIGN_URL_EXPIRY || 604800000;
```

Comment:
should parse env variable as int to avoid possibly type string with type int
```suggestion
    const preSignUrlExpiry = process.env.PRE_SIGN_URL_EXPIRY && !Number.isNaN(process.PRE_SIGN_URL_EXPIRY)
        ? Number.parseInt(process.env.PRE_SIGN_URL_EXPIRY) : 604800000;
```